swag/db.py

```diff
@@ -197,6 +251,63 @@ def creation_date(self, value):
         self.swagdb.creation_date[self.id] = value
 
 
+class Cagnotte:
+    def __init__(self, swagdb, cagnotte_id) -> None:
+        self.swagdb = swagdb
+        self.cagnotte_id = cagnotte_id
+
+    def get_info(self):
+        return CagnotteInfo(self)
+
+    @property
+    def cagnotte_name(self):
+        return self.swagdb.cagnotte_name[self.cagnotte_id]
+
+    @cagnotte_name.setter
+    def cagnotte_name(self, value):
+        self.swagdb.cagnotte_name[self.cagnotte_id] = value
+
+    @property
+    def cagnotte_balance(self):
+        return self.swagdb.cagnotte_balance[self.cagnotte_id]
+
+    @cagnotte_balance.setter
+    def cagnotte_balance(self, value):
```

Comment:
Isn't `balance` explicit enough?

Reply:
Actually, just remove `cagnotte_` from every property, it's just redondant

Reply:
Resolved in 5c0172519fd958594b28d5e474628e74e631957e